src/app/components/navigation/SharedToolbar.tsx

```diff
@@ -109,6 +140,19 @@ export const SharedToolbar = (props: SharedToolbarProps): JSX.Element => {
                 }}
                 {...other}
             >
+                {getIsFireworkHoliday() && (
+                    <div
+                        id="fireworks"
+                        style={{
+                            position: 'fixed',
+                            top: 0,
+                            left: 0,
+                            right: 0,
+                            height: sm ? 56 : 64,
```

Comment:
you could make this `bottom: 0` and then you won't have to worry about the switch from 64 to 56